Example/Emission/Placeholder.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Placeholder.swift
+//  Emission
+//
+//  Created by Luc Succes on 12/19/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
```

Comment:
Always remove these headers please. We have SCM for blaming and the copyright lines are harder to maintain then the single notice in the LICENSE file.